maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/CCOutcomeController.java

```diff
@@ -145,4 +150,20 @@ public ResponseEntity<List<RepOrderCCOutcomeDTO>> findByRepIdLengthInHeader(@Pat
         responseHeaders.setContentLength(service.findByRepId(repId).size());
         return ResponseEntity.ok().headers(responseHeaders).build();
     }
+
+    @Operation(description = "Deleting Crown Court Outcome by a RepOrder")
```

Comment:
I would consider amending the description to ensure it's clear the deletion is by Rep Order Id.
e.g.
`@Operation(description = "Deleting Crown Court Outcome by a Rep Order Id")`